prisma/schema.prisma

```diff
@@ -38,6 +38,7 @@ model Product{
     age Decimal
     category Category @relation(fields: [catSlug], references: [slug])
     catSlug String
```

Comment:
You can create multiple tables. Right now you store a lot of fields in one table. This is insecure